vpr/src/base/read_blif.cpp

```diff
@@ -116,6 +117,20 @@ struct BlifAllocCallback : public blifparse::Callback {
             set_curr_block(AtomBlockId::INVALID()); //This statement doesn't define a block, so mark invalid
         }
 
+       void assert_net_has_no_driver(std::string net_name) {
+            AtomNetId net_id = curr_model().create_net(net_name);
+            assert_net_has_no_driver(net_id, net_name);
+        }
```

Comment:
We can probably drop this overload if we restructure the subckt check slightly (see below).